catalyst/utils/distributed.py

```diff
@@ -0,0 +1,276 @@
+from collections import OrderedDict
+import copy
+import inspect
+from operator import itemgetter
+import os
+import socket
+import subprocess
+import sys
+from typing import Dict, Tuple
+
+import torch
+from torch import nn
+import torch.distributed
+from torch.utils.data import Dataset
+from torch.utils.data.distributed import DistributedSampler
+from torch.utils.data.sampler import Sampler
+
+from catalyst import utils
+from catalyst.utils.tools.typing import (
+    Criterion, Device, Model, Optimizer, Scheduler
+)
+
+__author__ = "Andrey Sheka"
+__maintainer__ = "Andrey Sheka"
+__email__ = "andrey.sheka@gmail.com"
```

Comment:
could you please remove the credentials, as this file consists of both already available Catalyst codebase and your new one?
Not sure if it correction option, nevertheless...